designer/client/src/components/graph/node-modal/fragment-input-definition/FragmentInputDefinition.tsx

```diff
@@ -14,13 +14,13 @@ interface Props extends Omit<MapVariableProps<FragmentInputParameter>, "readOnly
     isEditMode?: boolean;
 }
 
-export function useTypeOptions<Value = string>() {
+export function useTypeOptions<Value>(useDisplay = false) {
```

Comment:
Why do we need this `useDisplay` flag?

Reply:
See: https://github.com/TouK/nussknacker/pull/7167#discussion_r1850207954